### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

File: docs/source/setup.rst [hunk 1]
```diff
@@ -1,80 +1,110 @@
-Setup
-=====
+Authentication with Modelon Impact
+==================================
 
-This page describes the procedure to set up API keys to authenticate your login and
-various ways to use the generated API key to log in to the client.
+This page explains how to authenticate login to the Modelon Impact server from the Python client
```
Comment (efredriksson-modelon) [inline docs/source/setup.rst]: `authenticate login` does not sound right. Maybe `authenticate and login`?

**Review by efredriksson-modelon — CHANGES_REQUESTED** [review]
:+1:

**Comment by abhilash-kumar-nair** [comment]
@efredriksson-modelon Thanks for the review. Blame the AI that paraphrased my sentences :D

File: docs/source/authentication_with_modelon_impact.rst [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+When the token has been entered the first time, it will be stored at located in the directory
+``.impact`` in the users home folder and used in future requests and the prompt will not be 
+shown again. 
+
+Another option is to save the token in the environmental variable ``JUPYTERHUB_API_TOKEN``, which 
+would allow the client to authenticate without engaging in any interaction prompts.
+
+To view a list of active tokens or revoke active tokens, the user can
+go to the JupyterHub token manager (located at "https://impact.modelon.cloud/token").
```
Comment (iakovn) [inline docs/source/authentication_with_modelon_impact.rst:65]: I suggest using `https://impact.modelon.cloud/hub/token` everywhere for consistency (both work)

**Review by iakovn — APPROVED** [review]
Looks real nice! 🍎  Just a minor comment on using save URL

**Pull request merged**